student-info-system/schooladmin/tests/tests_major.py

```diff
@@ -38,28 +37,6 @@ def setUpTestData(cls):
         KLASS.m1.courses_required.add(KLASS.c3)
```

Comment:
are we using klass becuase class is a restricted keyword?

Reply:
sort of. It's a signal that it's not the real class, just shorthand. It's a variable we can sling around. So it's not just reserved, it's trying to signal "I'm intentional, not some automatic thing"